lib/client.js

```diff
@@ -355,8 +356,8 @@ Client.prototype.createTopics = function (topics, isAsync, cb) {
     cb = isAsync;
     isAsync = true;
   }
+  validateKafkaTopics(topics);
```

Comment:
When users call it as async method they will expect to catch errors in the callback. So this will end up being an uncaught exception which is unexpected. We should wrap it in a try catch and if it's `isAsync` is true call the callback with the error otherwise throw it again.